docs/work_tools/all.devops_docker.reference.md

```diff
@@ -257,3 +257,15 @@
 - This invokes a different service
 - To disable vim go to Settings -> Enable Vim Mode
 - We use the percent format for Jupytext
+
+## Docker Image Tags and Registries matrix
+
+| Image Tag           | AWS ECR | Dockerhub | GHCR |
+| ------------------- | ------- | --------- | ---- |
+| cmamp:dev           | [+]     | [-]       | [-]  |
```

Comment:
Can we explain what it means?
Is it where is updated? Or when we need to push the images?

Let's explain a bit more what and why